client/state/concierge/bookForm/reducer.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
```

Comment:
In https://github.com/Automattic/wp-calypso/pull/20979 I recommended that you probably don't even need to track timestamps in Redux. There's also undocumented data-layer features that let you track the status of a particular request without needing to save that status in state yourself. (I'm still trying to digest it a bit, but if you [add data-layer tracking  metadata to an action](https://github.com/Automattic/wp-calypso/blob/master/client/state/activity-log/actions.js#L89-L93) the request's info will be stored in `redux.state.dataRequests` and then by using [the `getRequest()` selector](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-request.js) and passing in the same action that generated the request, you can check `hasLoaded` for whether the request has gone through.

If there's still reason to keep this state around, it should go under `state.concierge.signupForm` with the other appointment data. There's no need to keep it in a separate state key, it's all data related to the multi-step signup form.

Reply:
Great feedback. Thanks for taking the time.

I looked over the data-layer metadata but could not figure out an easy way to use `getRequest` with an action that has a dynamic timestamp. For now I moved the status of the booking in the `signupForm`.